app/src/main/res/layout/view_edit_servings.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
```

Comment:
This `RelativeLayout` is useless. Much like myself, it has no id, no style, and no background.

Unless you're trying to horizontally center its child, but if you don't have a container than Android doesn't inflate the LinearLayout's layout params correctly because you don't have a parent view for the dialog. In which case, use a `FrameLayout` here because it's faster.